src/featureMatching/getIntensityCentroid.ts

```diff
@@ -5,6 +5,7 @@ import { getIntensityMoment } from './getIntensityMoment';
 
 /**
  * Compute the intensity centroid of an image for each channel.
+ * DOI: 10.1109/ICCV.2011.6126544
```

Comment:
There's no publicly-available reference?

Reply:
I guess a wiki link is better?

Reply:
Maybe keep both?

Reply:
Okay!

Reply:
Don't you agree that we should share the doi of https://doi.org/10.1006/cviu.1998.0719 (source of the information) ?

Reply:
> It seems that the actual reference is https://doi.org/10.1006/cviu.1998.0719

Woops I'm really sorry, I had actually missed that message and got confused.